pkg/scheduler/util/utils.go

```diff
@@ -59,7 +59,7 @@ func GetPodStartTime(pod *v1.Pod) *metav1.Time {
 func GetEarliestPodStartTime(victims *extenderv1.Victims) *metav1.Time {
 	if len(victims.Pods) == 0 {
 		// should not reach here.
-		klog.ErrorS(fmt.Errorf("victims.Pods is empty. Should not reach here"), "")
+		klog.Background().Error(nil, "victims.Pods is empty. Should not reach here")
```

Comment:
@pohly are you ok going with the general rule that if a line shouldn't be reached we can use Background?

I guess we rather not use `panic`

Reply:
Yes, that seems reasonable.